packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -100,16 +100,9 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
 
   /**
    * Optional flag to render the Slider as disabled.
-   * @defaultvalue false
    */
   disabled?: boolean;
 
-  /**
-   * Optional flag to decide that thumb will snap to closest value while moving the slider
-   * @defaultvalue false
-   */
-  snapToStep?: boolean;
```

Comment:
You can't just remove a prop from the API contract. This PR can not go with a change like this.

Reply:
oh I see what you mean just added snapping back in